src/cli/main.rs

```diff
@@ -198,6 +204,15 @@ fn format_string(
     }
 }
 
+fn get_ignore(directory: &Path) -> Result<Gitignore> {
+    let file_path = find_ignore_file_path(directory.to_path_buf(), true).unwrap();
```

Comment:
Should we always search recursively here? Or only when `--search-parent-directories` is specified?

Reply:
Addressed.